tools/opcode_metadata/generate_py_opcode_metadata.py

```
@@ -0,0 +1,107 @@
+"""
+Generate Lib/_opcode_metadata.py for RustPython bytecode.
+
+This file generates opcode metadata that is compatible with CPython 3.14.
+"""
+
+import functools
+import io
+import itertools
+import operator
+import pathlib
+import typing
```

Flake8 (7.3.0)

[error] 'typing' imported but unused

(F401)